test/doc/algo/transform_copy_if.cpp

```diff
@@ -0,0 +1,31 @@
+#include <eve/module/core.hpp>
+#include <eve/module/algo.hpp>
+#include <tts/tts.hpp>
+#include <iostream>
+#include <ranges>
+#include <vector>
+
+int main()
+{
+  std::vector<int> input(16);
+  eve::algo::iota(input, 0);
+  std::cout << "Input vector:\n  " << tts::as_string(input) << "\n\n";
+
+  auto func = [](eve::wide<int> e) {
```

Comment:
it's eve::nofs_wide unfortunately. That's a separate can of worms - just use eve::like<int> auto.